1_The_Physical/Guess_Point_Level_1_Stateless/2_Arrange_Act_Assert/2_1_Password_Validator/src/index.ts

```diff
@@ -0,0 +1,19 @@
+export class PasswordValidator {
+  public static check(input: string) {
+    const isBetweenFiveAndFifteen = input.length >= 5 && input.length <= 15;
```

Comment:
Expressive, declarative looking production code at the right level of abstraction. Great